scheme/common/cca/realm/realm_handlers.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/veraison/services/handler"
+	"github.com/veraison/services/log"
+)
+
+func SynthKeysForCcaRealm(scheme string, tenantID string,
+	refVal *handler.Endorsement,
+) ([]string, error) {
+	rim, err := GetRIM(refVal.Attributes)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get rim %w", err)
+	}
+	lookupKey := RealmRefValLookupKey(scheme, tenantID, rim)
+	log.Debugf("Scheme %s Plugin Reference Value Look Up Key= %s\n", scheme, lookupKey)
```

Comment:
consider dropping this line

Reply:
This debug was `extremely useful while debugging issues, I have removed almost all the un-necessary prints, but this one is particularly helpful!

Reply:
Need to have some tracing from Front End to Back!!! Concurrent Requests across all the pipeline!!

Reply:
See issue #236